TrafficTracker.py

```diff
@@ -86,15 +102,18 @@ def get_new_video_capture(self, video_path):
         self.vid_width = int(self.vid.get(cv2.CAP_PROP_FRAME_WIDTH))
         self.vid_height = int(self.vid.get(cv2.CAP_PROP_FRAME_HEIGHT))
         self.vid_fps = int(self.vid.get(cv2.CAP_PROP_FPS))
+        self.length = int(self.vid.get(cv2.CAP_PROP_FRAME_COUNT))
+
 
-    def draw_boxes(self, img, bbox, identities=None):
+    def draw_boxes(self, img, bbox, classes, identities=None):
         for i, box in enumerate(bbox):
             x, y, w, h = [int(i) for i in box]
             # box text and bar
             id = int(identities[i]) if identities is not None else 0
+            cl = classes[i]
             color = self.compute_color_for_labels(id)
-            label = '{}{:d}'.format("", id)
-            t_size = cv2.getTextSize(label, cv2.FONT_HERSHEY_PLAIN, 2, 2)[0]
```

Comment:
This is where I halved the thickness of the bounding box and text

We should probably parameterize these, maybe with a global var set at the top or with a cli flag